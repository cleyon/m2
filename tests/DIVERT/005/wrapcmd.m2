@newcmd hi
Hello, world!
@endcmd
@wrap @hi
Start of file
End of file
