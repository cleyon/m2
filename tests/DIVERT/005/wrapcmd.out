Start of file
End of file
Hello, world!
